training/common/driver-toolkit/Containerfile

```diff
@@ -7,8 +7,7 @@ ARG ENABLE_RT=''
 
 USER root
 
-RUN dnf -y update \
-    && dnf info --installed kernel > /dev/null || dnf install -y kernel-core \
+RUN dnf info --installed kernel > /dev/null || dnf install -y kernel-core \
```

Comment:
How about dnf -y update --exclude kernel\*

Reply:
Works perfectly, I changed the code and the commit message :+1: